saleor/core/taxes.py

```diff
@@ -22,6 +22,10 @@ def zero_taxed_money(currency: str) -> TaxedMoney:
     return TaxedMoney(net=zero, gross=zero)
 
 
+def identical_taxed_money(money: Money) -> TaxedMoney:
```

Comment:
Maciek suggested renaming it, let's look for other options / ask others for feedback.

Reply:
I went with Maciek's suggestion, maybe it actually is clearer